dnn/model/yolov3-tiny.py

```diff
@@ -0,0 +1,248 @@
+from absl import flags
+from absl.flags import FLAGS
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras import Model
+from tensorflow.keras.layers import (
+    Add,
+    Concatenate,
+    Conv2D,
+    Input,
+    Lambda,
+    LeakyReLU,
+    MaxPool2D,
+    UpSampling2D,
+    ZeroPadding2D,
+    BatchNormalization,
+)
+from tensorflow.keras.regularizers import l2
+from tensorflow.keras.losses import binary_crossentropy
+
+flags.DEFINE_integer('yolo_max_boxes', 100, 'maximum number of boxes per image')
```

Comment:
100 seems a lot...

Reply:
To be honest, I do not know how many is appropriate, so I left it here as a hyper parameter what we should tune.